### ISSUE TITLE
[BUG] Shift.BY Issue

### ISSUE THREAD

**Issue by MadSciSlack** [Kitteh6660/DramaticDoors#87]
**Important Information**
Minecraft Version: 1.19.2
Mod Loader System: Forge
DramaticDoors Version: 3.1.5
Development server environment.

**Describe the bug**
@Inject(@At("INVOKE")) Shift.BY=2 on forge-dramaticdoors_forge.mixins.json:OpenDoorsTaskMixin::handler$cal000$injectStartP1 exceeds the maximum allowed value: 0. Increase the value of maxShiftBy to suppress this warning.

This warning spam has been present for as long as I've used the mod.  Is there any way to correct this?

**Expected behavior**
Warnings corrected eventually.

Thanks.

**Comment by Kitteh6660** [comment]
It's strange. I've even specified the max shift in mixins.json but for some reason, it's not even recognized.

**Comment by ShetiPhian** [comment]
I found this while hunting for the solution myself.
Looks like max shift goes into the injectors block.
~~~
"injectors": {
    "defaultRequire": 1,
    "maxShiftBy": 2
}
~~~

**Comment by Kitteh6660** [comment]
I will give it a try, thanks!

**Comment by Kitteh6660** [comment]
Should be resolved in mod version 3.2.3.
